app/views/content/home/_salaries-banner.html.erb

```diff
@@ -2,6 +2,6 @@
   <%= render CallsToAction::SimpleComponent.new(icon: "icon-money") do %>
     <h3>Earn at least &pound;30k a year</h3>
     <p>All qualified teachers will have a <strong>starting salary of at least £30,000</strong> (or higher in London).</p>
-    <p>Find out more about the <%= link_to("salaries and benefits of teaching", page_path("salaries-and-benefits")) %>.</p>
+    <p>Find out more about <%= link_to("teacher pay and the benefits of teaching", page_path("is-teaching-right-for-me/teacher-pay-and-benefits")) %>.</p>
```

Comment:
I know this is in prod already but I'm not sure this link text is quite right - this implies that the destination talks about the benefits of teaching?

Is it possible to reword it somehow? Eg
Find out more about teacher pay and benefits. (Is this too short?)